test/integration/034_redshift_test/test_backup_table_option.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from test.integration.base import DBTIntegrationTest, use_profile
+
+
+class TestBackupTableOption(DBTIntegrationTest):
+    @property
+    def schema(self):
+        return 'backup_table_option_034'
+
+    @staticmethod
+    def dir(path):
+        return os.path.normpath(path)
+
+    @property
+    def models(self):
+        return self.dir("models_backup")
+
+    @property
+    def project_config(self):
+        return {
+            'config-version': 2,
+            'data-paths': [self.dir('seed')],
+            'seeds': {
+                'quote_columns': False,
+            }
+        }
+
+    def project_config_append(self, new_name, new_element):
+        self._project_config[new_name] = new_element
+
+    def check_backup_param_template(self, test_table_name, backup_is_expected):
+        # Use raw DDL statement to confirm backup is set correctly on new table
+        with open('target/run/test/models_backup/{}.sql'.format(test_table_name), 'r') as ddl_file:
```

Comment:
I know this is a pattern present in other integration tests (I see it in 020, 028, 029). I still hesitated about this, because the run file path (`target/run/.../*.sql`) is not _exactly_ contracted. After thinking about it more, this would still be a highly visible user-facing change, and so if we were to ever change it, this test failure would be a nice way to call that out. Just leaving this comment for our future reference.